lib/SimplePie/SimplePie/Enclosure.php

```diff
@@ -627,7 +627,7 @@ public function get_link()
 	{
 		if ($this->link !== null)
 		{
-			return urldecode($this->link);
+			return $this->link;
```

Comment:
This line was introduced in https://github.com/simplepie/simplepie/commit/8a5f6013c4405cc8d5ccb27f2d76c78e8657df53#diff-a710c236cc7775672edbe5840d8fb96b0e91b0d56eff709f6eace2da47c1fe43R349-R359

Seems to be a similar error than https://github.com/simplepie/simplepie/commit/efb1d8e34c39276fb88fff0dcff01c673bec7e18

Reply:
Upstream PR https://github.com/simplepie/simplepie/pull/768